dev/com.ibm.ws.install/src/com/ibm/ws/install/internal/cmdline/ExeInstallAction.java

```diff
@@ -175,6 +175,10 @@ private ReturnCode initialize(Arguments args) {
                 logger.log(Level.SEVERE, Messages.INSTALL_KERNEL_MESSAGES.getLogMessage("ERROR_TOOL_DIRECTORY_NOT_EXISTS", localDir.getAbsolutePath()));
```

Comment:
I would suggest to merge the conditions together and print out the same message.

            if (!localDir.isDirectory() && !localDir.exists()) {
                logger.log(Level.SEVERE, Messages.INSTALL_KERNEL_MESSAGES.getLogMessage("ERROR_TOOL_DIRECTORY_NOT_EXISTS", localDir.getAbsolutePath()));
                return ReturnCode.IO_FAILURE;
            }